app/main.py

```diff
@@ -1,2 +1,65 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    capacity = 8
+    load_factor = 2 / 3
+    size = 0
+
+    def __init__(self) -> None:
+        self.hash_table = [[None, None]] * (self.capacity + 1)
+
+    def __len__(self) -> int:
+        return self.size
+
+    def tst_ind(self, ar_ind: int) -> int:
+        ar_ind = ar_ind + 1
+        if ar_ind > self.capacity:
+            ar_ind = 0
+        return ar_ind
```

Comment:
Pay more attention on naming ( functions , viriables etc ) tst_ind ?? ar_ind ?? It is not descriptive ( Fix that please